tools/setup/emoji/emoji_names.py

```diff
@@ -1580,7 +1594,7 @@
     "1f9c8": {"canonical_name": "butter", "aliases": ["dairy"]},
     "1f9c9": {"canonical_name": "mate", "aliases": []},
     "1f9ca": {"canonical_name": "ice", "aliases": ["ice_cube", "iceberg"]},
-    "1f9cb": {"canonical_name": "bubble_tea", "aliases": ["bubble"]},
+    "1f9cb": {"canonical_name": "bubble_tea", "aliases": []},
```

Comment:
Same question with this one.

Reply:
presumably the same with "eye_in_speech_bubble"